src/components/pages/Groups/RolesPanelForm/RolesPanelForm.tsx

```diff
@@ -36,6 +39,25 @@ const RolesPanelForm = ({
   removeRoles,
   setRemoveRoles,
 }: Props) => {
+  const { data } = useGetRoles();
```

Comment:
This is just a partial implementation of selecting roles from the API to unblock this and other work.